api/schema/aws_account_resource.go

```diff
@@ -66,6 +66,33 @@ var (
 						Mode: ImmutableAttributeMode,
 					},
 				},
+				"role_arn": StringResourceAttributeWithMode{
```

Comment:
What is the point of setting the role ARN? We don't seem to do anything with that on the server side.

Reply:
Nevermind. It is indeed necessary.

Reply:
we are storing the arn in vault. and are using it to do aws queries. also facilitates usage of existing roles.